social-media/video-scripts/billionaire-theory-of-change-3min.md

```
@@ -0,0 +1,120 @@
+# Theory of Change — 3-Minute Billionaire Pitch
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)